app/tut/ga/reamhra.md

```diff
@@ -1,9 +1,93 @@
 ---
-title: Réamhra Teagaisc
+title: Réamhrá Teagaisc
 ---
 
 # Fáilte!
 
-**Fáilte go dtí Teagasc Setanta**.
+**_Fáilte go dtí an Teagasc Setanta_**.
 
-Tá súil againn go bhfuil tú réidh chun foghlaim gach rud faoi ríomhchlárúchán, agus conas a oibríonn é agus an Ghaeilge le chéile.
+Tá súil againn go bhfuil tú réidh chun foghlaim gach rud faoi [[ríomhchlárúchán|programming]], agus
+conas a oibríonn é agus an Ghaeilge le chéile.
+
+Níl ach cúpla rudaí le míniú sula mbeidh tú réidh le tosú.
+
+## Aistriúcháin
+
+Má fheiceann tú focal le folíne mar [[seo|D'aimsigh tú mé!]], is focal é le
+[[aistriúchán|translation]].  Cuir do luch thar an focal nó cliceáil air chun a aistriúchán a
+fheiceáil. [[Bain triail as!|Try it out!]]
+
+## Eagarthóirí Setanta
+
+Ar fud an [[teagasc|tutorial]] seo, aimseoidh tú [[eagarthóirí|editors]] Setanta mar seo:
+
+{{{
+>-- Is eagarthóir Setanta é seo
+scríobh("Dia duit!")
+}}}
+
+Is féidir linn eagarthóirí a úsáid chun ríomhchláir Setanta a scríobh. Clóscríobhaimid ríomhchlár
+isteach sa bosca ar chlé agus brúimid <iron-icon class="play" icon="av:play-arrow"></iron-icon> chun
+é a tosú. Beidh torthaí an ríomhchláir le feiceáil ar dheis.
+
+Brúigh an cnaipe <iron-icon class="play" icon="av:play-arrow"></iron-icon> chun an cód san
+eagarthóir a rith.  Bá chóir go bhfuil "Dia duit!" scríofa ar an taobh dheis.
+
+## An Consól
+
+Ar an taobh dheis, ba chóir go bhfuil seo le feiceáil:
+
+> Dia Duit!
+
+Is é seo an consól. Is féidir linn an consól a úsáid chun [[teachtaireachtaí|messages]] a léamh agus
+a scríobh.  Sa ríomhchlár roimhe (`scríobh("Dia duit!")`{.setanta}), d'úsáideamar an
+[[gníomh|action]] `scríobh`{.setanta} chun "Dia duit!" a scríobh amach ar an gconsól.
+
+Freisin, is féidir linn an consól a úsáid chun teachtaireachtaí a sheoladh chuig an ríomhchlár. Bain
+triail as an gcéad ríomhchlár eile seo a rith; Ba chóir go scríobhfadh sé "Cad is ainm duit?" sa
+chonsól. Chlóscríobh d'ainm isteach sá bosca téacs ag an bun agus brúigh an
+[[eochair iontrála|enter key]].
+
+{{{
+ainm := ceist("Cad is ainm duit?")
+scríobh("Dia duit", ainm)
+}}}
+
+Brúigh an <iron-icon class="clear" icon="icons:clear"></iron-icon> chun an consól a ghlanadh.
+
+### Taispeántas
+
+![Ag scríobh d'ainm](../en/assets/intro-name.gif)
+
+## An Stáitse
+
+Tá dhá táib i ngach eagarthóir, agus is féidir leat athraigh eatarthu. Taispeánann ceann amháin an
+consól, agus taispeánann ceann eile an stáitse.
+
+Is féidir linn an stáitse a úsáid le Setanta chun pictiúir nó [[beochain|animations]] a tharraing,
```

Comment:
An bhfuil `beochain` ceart, níor aimsigh mé an t-iolra sa fóclóir.

Reply:
beochaintí dar le téarma.ie 
https://www.tearma.ie/q/Beochan%20/

Reply:
Is féidir leat athrú eatarthu

Reply:
A tharraingt